api/python/provisioner/commands/validate_setup.py

```diff
@@ -0,0 +1,177 @@
+#
```

Comment:
Just call this validate.
We'll park multiple classes under this header.
Also, change path to `api/python/provisioner/commands/validators/`
- node.py
- network.py
- storage.py
- release.py

Reply:
Looked like the PR was getting too long with configure and validation. So, to restrict the scope, have created a ticket and will be addressing module movement in it. https://jts.seagate.com/browse/EOS-18518